File: zsbApp.PreviewPdfByFoxit/FoxitPDFViewerForNET200_32914_enu/Examples/Visual Studio 2010_2012_2013_2015/CSharp/HiddenToolbars/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace HiddenToolbars
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            // In the form constructor do not use any Viewer methods (Open, Print etc.) that perform UI activities.
        }

        private void Open_Click(object sender, EventArgs e)
        {
            if (this.openFileDialog1.ShowDialog(this.pdfViewer1) == DialogResult.OK)
            {
                //Calling  method to open the selected document from file dialog.
                this.pdfViewer1.Open(this.openFileDialog1.FileName);
            }
          
        }

        private void Print_Click(object sender, EventArgs e)
        {
            //Calling method for print.
            this.pdfViewer1.Print();
        }

        private void Close_Click(object sender, EventArgs e)
        {
            // Calling mehtod to close the current document.
            this.pdfViewer1.Close();
        }

    }
}

File: zsbApp.PreviewPdfByFoxit/FoxitPDFViewerForNET200_32914_enu/Examples/Visual Studio 2010_2012_2013_2015/VB/HiddenToolbars/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Public Class Form1
    Public Sub Form1()
        InitializeComponent()
        ' In the form constructor do not use any Viewer methods (Open, Print etc.) that perform UI activities.
    End Sub

    Private Sub Open_Click(ByVal sender As System.Object, ByVal e As EventArgs)
        If (Me.openFileDialog1.ShowDialog(Me.pdfViewer1) = DialogResult.OK) Then
            'Calling  method to open the selected document from file dialog.
            Me.pdfViewer1.Open(Me.openFileDialog1.FileName)
        End If
    End Sub
    Private Sub Print_Click(ByVal sender As System.Object, ByVal e As EventArgs)
        'Calling method for print.
        Me.pdfViewer1.Print()
    End Sub
    Private Sub CloseDoc_Click(ByVal sender As System.Object, ByVal e As EventArgs)
        ' Calling mehtod to close the current document.
        Me.pdfViewer1.Close()
    End Sub

End Class
